pkg/cmd/run.go

```diff
@@ -243,6 +243,10 @@ func (o *runCmdOptions) validate() error {
 		return err
 	}
 
+	if o.OutputFormat != "" && o.Dev {
+		return errors.New("option '-o' is not compatible with '--dev'")
```

Comment:
We better keep using the `fmt` package as we're doing for the rest of errors, ie `return fmt.Errorf(invalid label specification %s. Expected "<labelkey>=<labelvalue>", label)`

Reply:
Ok I'd use the `fmt` package